File: multiple.py
def an(a1,n,d):
    return a1+(n-1)*d
def sn(a1,n,an):
    return int(n/2*(a1+an))
m = int(input())
a1,d,n = 3,3,m//3
am = an(a1,n,d)
s3 = sn(a1,n,am)
a1,d,n = 5,5,m//5
am = an(a1,n,d)
s5 = sn(a1,n,am)
a1,d,n = 15,15,m//15
am = an(a1,n,d)
s15 = sn(a1,n,am)
print(s3+s5-s15)

File: multiple.cpp
#include<iostream>
using namespace std;
int an(int a1,int n,int d)
{
    return a1+(n-1)*d;
}
int sn(int n,int a1,int an)
{
    return n/2.0*(a1+an);
}
int main()
{
    int m;
    cin >> m;
    int a1,am,n,d;
    a1=3;
    d=3;
    n=m/3;
    am = an(a1,n,d);
    int s3 = sn(n,a1,am); 

    a1=5;
    d=5;
    n=m/5;
    am = an(a1,n,d);
    int s5 = sn(n,a1,am); 

    a1=15;
    d=15;
    n=m/15;
    am = an(a1,n,d);
    int s15 = sn(n,a1,am); 

    cout << s3+s5-s15;
}